src/sdk/sdk.ts

```diff
@@ -87,9 +88,16 @@ export class FusionSDK {
             throw new Error('quoter has not returned quoteId')
         }
 
+        const nonceManager = NonceManager.new({
+            maker: params.walletAddress,
+            blockchainProvider: this.config.blockchainProvider
+        })
+
+        const nonce = await nonceManager.getNonce()
```

Comment:
add nonce: OrderNonce | string | number to `OrderParams`. And request nonce only on Auto mode